rest-api-model-ql/src/main/kotlin/org/modelix/sample/restapimodelql/LightModelClientWrapper.kt

```diff
@@ -0,0 +1,120 @@
+package org.modelix.sample.restapimodelql
+
+import University.Schedule.*
+import io.ktor.client.*
+import io.ktor.client.engine.cio.*
+import io.ktor.client.plugins.websocket.*
+import jetbrains.mps.lang.core.N_BaseConcept
+import kotlinx.coroutines.delay
+import org.modelix.client.light.LightModelClient
+import org.modelix.client.light.WebsocketConnection
+import org.modelix.metamodel.typed
+import org.modelix.model.repositoryconcepts.N_Module
+import org.modelix.model.repositoryconcepts.N_Repository
+import org.modelix.model.repositoryconcepts.models
+import org.modelix.model.repositoryconcepts.rootNodes
+import org.modelix.model.server.api.buildModelQuery
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import kotlin.time.Duration.Companion.milliseconds
+import kotlin.time.Duration.Companion.seconds
+
+val logger: Logger = LoggerFactory.getLogger("org.modelix.sample.restapimodelql.ModelServerLightWrapper")
+
+class LightModelClientWrapper {
+
+    private var MPS_MODEL_NAME: String = ""
+    private var WS_CONNECTION: String
```

Comment:
The upper case namings pretend to be constants, but with `var` they are definitely not:

```suggestion
    private var mpsModelName: String = ""
    private var wsConnection: String
```

Reply:
And why are vars required at all? Things are filled in the constructor. Can't they be `val`s?